x-pack/plugins/apm/server/lib/helpers/get_internal_saved_objects_client.ts

```diff
@@ -0,0 +1,16 @@
+/*
```

Comment:
smaller helper function to get the internal saved objects client from the core setup object since we do this in a few places.